src/commonControls/questionnaireControl/QuestionnaireControl.ts

```diff
@@ -602,9 +614,10 @@ export class QuestionnaireControl extends Control implements InteractionModelCon
             },
             apl: {
                 enabled: true,
-                askQuestion: QuestionnaireControlAPLPropsBuiltIns.defaultAskQuestion({
+                askQuestion: QuestionnaireControlAPLPropsBuiltIns.DefaultAskQuestion({
                     title: undefined, // the default is wired up in defaultAskQuestion()
```

Comment:
nit: the default is wired up in  _`defaultAskQuestion()`_. ---> `DefaultAskQuestion()`